### PR TITLE
ZK-4077: sizedByContent and sizabled trigger delayed rerender breakin…

### PR DESCRIPTION
**Pull request by DevChu**
…g UI on next event

### CODE REVIEW

File: zul/src/archive/web/js/zul/mesh/HeadWidget.js [hunk 1]
```diff
@@ -119,6 +119,7 @@ zul.mesh.HeadWidget = zk.$extends(zul.Widget, {
 		owner._innerWidth = headWidth || owner.eheadtbl.width || owner.eheadtbl.style.width;
 		owner.fire('onInnerWidth', owner._innerWidth);
 		owner.fireOnRender(zk.gecko ? 200 : 60);
+		owner.forcedClearAutoSizing_(); // clear span and sizedByContent without rerender
```
Comment (scribetw) [inline zul/src/archive/web/js/zul/mesh/HeadWidget.js]: disableAutoSizing_

File: zul/src/org/zkoss/zul/impl/HeadersElement.java [hunk 2]
```diff
@@ -73,8 +73,6 @@ protected void renderProperties(org.zkoss.zk.ui.sys.ContentRenderer renderer) th
 	public void service(org.zkoss.zk.au.AuRequest request, boolean everError) {
 		final String cmd = request.getCommand();
 		if (cmd.equals(ZulEvents.ON_COL_SIZE)) {
-			((MeshElement) this.getParent()).setSpan(false); //clear span
```
Comment (scribetw) [inline zul/src/org/zkoss/zul/impl/HeadersElement.java]: setSpanDirectly

**Comment by charlesqqq** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**